### PR TITLE
Add trust_input option to postgresql_slot module

### PR DESCRIPTION
**Pull request by andytom**
##### SUMMARY

Have added a trust_input option to the postgresql_slot module. This
only checks the session_role since all other options are passed as
parameters.

Related to  #106 and #210.
##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

`plugins/modules/database/postgresql/postgresql_slot.py`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by andytom** [comment]
ready_for_review

File: plugins/modules/database/postgresql/postgresql_slot.py [hunk 1]
```diff
@@ -70,6 +70,11 @@
     - Permissions checking for SQL commands is carried out as though
       the session_role were the one that had logged in originally.
     type: str
+  trust_input:
+    description:
+    - If C(no), check whether values of some parameters are potentially dangerous.
```
Comment (Andersson007) [inline plugins/modules/database/postgresql/postgresql_slot.py:75]:
```suggestion
    - If C(no), check whether a value of I(session_role) is potentially dangerous.
    - It does make sense to use C(yes) only when SQL injections via I(session_role) are possible.
```
Considering that only `session_role` will be checked, i suggest pointing this out explicitly.
And we could put this explanations into all next modules in such cases where the parameter can be used for checking a small number of their other parameters.
What do you think?

Comment (andytom) [reply plugins/modules/database/postgresql/postgresql_slot.py]: Makes perfect sense to me, have updated the docs.

**Comment by Andersson007** [comment]
@andytom thanks for the contribution!
merged into master

**Pull request merged**